### PR TITLE
[DOCS-5928] Copy edited the EC-S3 plugin for BeeWrite standards

### PR DESCRIPTION
**Pull request by kellie-freeman**
Copy edited the EC-S3 plugin for BeeWrite standards

### CODE REVIEW

File: help/help.adoc [hunk 1]
```diff
@@ -1,448 +1,301 @@
...
 [cols="1a,1a",options="header"]
 |===
 |Parameter |Description
-|Configuration |The name of the configuration which holds all the connection information. This must reference a valid existing configuration. (Required)
-|Bucket Name |Name of the bucketof which to list the folders. (Required)
+|Configuration |Required. The name of the configuration that holds all the connection information. This must reference a valid existing configuration.
+|Bucket Name |Required. Name of the bucket in which to list the folders.
 |Folder Name |Name of the folder or prefix to include in the list.
-|List Objects in this folder or Include all sub folders? |If selected, all objects in this folder and all subfolders will be in the list.
+|List Objects in this folder or Include all sub folders? |If selected, all objects in this folder and all sub-folders are included in the list.
```
Comment (jsmith-cb) [inline help/help.adoc:173]:
```suggestion
|List Objects in this folder or Include all subfolders? |If selected, all objects in this folder and all subfolders are included in the list.
```
I don't think this should be hyphenated.  Refer to https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sub. Also, there is a space in `sub folders` in the label.

File: help/help.adoc [hunk 2]
```diff
@@ -1,448 +1,301 @@
...
+==== WebsiteHosting input parameters
 
+.WebsiteHosting input parameters
 [cols="1a,1a",options="header"]
 |===
 |Parameter |Description
 |Configuration |The name of the configuration that has all the connection information. This must refer to a valid existing configuration.
 |Bucket Name |Name of the bucket to create.
-|Enable website hosting |After you enable your bucket for static website hosting, all your content is accessible to web browsers through the Amazon S3 endpoint for your bucket.
+|Enable website hosting |If selected, after you enable your bucket for static website hosting, all of your content is accessible to web browsers through the Amazon S3 endpoint for your bucket.
```
Comment (jsmith-cb) [inline help/help.adoc:226]:
```suggestion
|Enable website hosting |If selected, after you enable your bucket for static website hosting, your content is accessible to web browsers through the Amazon S3 endpoint for your bucket.
```

**Review by jsmith-cb — APPROVED** [review]
For `|Bucket Name |`, you switch back and forth between using `where` and `in which`. I don't think it is that big of a deal, but for consistency, you may want to harmoinze.

**Review by alexey-sinyuk — APPROVED** [review]

File: help/help.adoc (hunk 1 above)
Comment (kellie-freeman) [reply help/help.adoc:173]: Yeah, I was debating this change as well. Thanks!

**Pull request merged**